app/views/microposts/_micropost.html.erb

```diff
@@ -11,5 +11,16 @@
 			<%= link_to "delete", micropost, method: :delete,
 											 data: { confirm: "Вы уверены?" } %>
 		<% end %>
+		<!-- I know it wery ugly and this code should be in model or in helper (I guess) -->
+		<% arr = micropost.votes.collect {|i| i[:grades].to_f } %>
+		<p> <%= arr == [] ? 'No votes' : arr.inject{ |sum, el| sum + el }.to_f / arr.size %> </p>
 	</span>
+    <% if voted(micropost.votes) == false %>
+    <!-- You will probably laugh at my voting form)) -->
```

Comment:
Self-deprecating comments are really hurtful to read. 
If you can't take pride in your achievements, however small they may seem, you are not going to have a good time.